src/Components/CreateImageWizard/steps/Users/component/UserInfo.tsx

```diff
@@ -115,6 +130,34 @@ const UserInfo = () => {
           name="user Administrator"
         />
       </FormGroup>
+      <EmptyState variant={EmptyStateVariant.lg}>
+        <EmptyStateFooter>
+          <Popover
+            hasAutoWidth
+            maxWidth="35rem"
+            bodyContent={
+              <TextContent>
+                <Text>
+                  If you regret and do not want to add a user it is totally
+                  fine, just click the button to remove it
+                </Text>
+              </TextContent>
+            }
+          >
+            <Button
+              variant="plain"
+              aria-label="Activation key popover"
+              aria-describedby="subscription-activation-key"
+              className="pf-v5-u-pl-sm pf-v5-u-pt-0 pf-v5-u-pb-0 pf-v5-u-pr-0"
+            >
+              <HelpIcon />
+            </Button>
+          </Popover>
+          <Button variant="secondary" onClick={onRemoveUserClick}>
+            Remove a user
+          </Button>
+        </EmptyStateFooter>
+      </EmptyState>
```

Comment:
Nice! I think a remove button will be a good addition.

Let's make it simpler - we already have an empty user state and there's not really a reason to add another one. I'd also drop the popover, "Remove user" as a button text is self-explanatory enough.

Reply:
How about something like one of these?

![image](https://github.com/user-attachments/assets/62080688-9823-4c05-ad6e-6a8ff3cd7f41)

The first one is a "plain" button. The second is tertiary with an icon and a tooltip.

Reply:
I implement the button with the tooltip. if I remove the emptyState, the rectangle of the button looks wired